cxx/gendb.cc

```diff
@@ -60,6 +60,9 @@ void GenDB::incorporate(
     // Incorporate the items/value into the query relation.
     incorporate_query_relation(prng, query_rel, items, val);
   }
+
+  // Add to the record_class's CRP.
```

Comment:
I don't think we should have a CRP for the Record class. What is the problem, and is there another way to fix it?

Reply:
Well, the specific problem was that I assumed that there would be a CRP for the record class, and I wrote a test under that assumption, and then I added these lines to make that test pass.

But I also use the record class's CRP in pclean_lib.cc::make_pclean_sample to create a class_item for each new row.  I'm open to suggestions for better ways to implement that, but I believe that sampling from the record class's CRP is what the model spec says to do.

Reply:
For make_pclean_sample, I think we should just use a counter for class_item, or have the function take a vector of unique row IDs. I don't think the spec says to sample from the record class's CRP -- there's a one-to-one correspondence between observations and record class entities, and sampling from a record class CRP would result in multiple observations of a single record entity.

Reply:
Done.  When I was talking about the spec, I was talking about the generative model described on page 7 under "PClean entity model".